### PR TITLE
Add initial source code

### PR DESCRIPTION
**Pull request by Peque**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.opencm3/
```
Comment (cua-cua) [inline .gitignore]: I would include on documentation from where and how to get opencm3 library to compile the code. Maybe a new issue for that?

Comment (Peque) [inline .gitignore]: Done (#22). :blush:

File: src/main.c [hunk 2]
```diff
@@ -1,43 +1,72 @@
...
+ *
+ * Enable required clocks for the GPIOs as well.
+ */
+static void setup_clock(void)
 {
-	/* Set STM32 to 72 MHz. */
-	rcc_clock_setup_in_hse_12mhz_out_72mhz();
-
-	/* Enable GPIOC, and AFIO clocks. */
+	rcc_clock_setup_in_hse_8mhz_out_72mhz();
```
Comment (cua-cua) [inline src/main.c]: Why are you using now 8 mhz as input instead of the 12 mhz from previous commit?

File: src/main.c [hunk 3]
```diff
@@ -1,43 +1,72 @@
...
-	AFIO_MAPR |= AFIO_MAPR_SWJ_CFG_FULL_SWJ_NO_JNTRST;
 
-	/* GPIO initialization */
-	gpio_clear(GPIOC, GPIO13);
+/**
+ * @brief Set SysTick interruptions frequency and enable SysTick counter.
+ *
+ * SYSCLK is at 72 MHz as well as the AHB (by default, the AHB divider is set
+ * to 1, so the AHB output has the same frequency as the SYSCLK).
+ */
```
Comment (cua-cua) [inline src/main.c]: Could you add here some reference information about AHB divider?

File: .gitignore (hunk 1 above)
Comment (cua-cua) [inline .gitignore]: Thank you!

File: src/main.c (hunk 2 above)
Comment (Peque) [inline src/main.c]: Our board uses an 8 MHz external clock. :blush:

File: src/main.c (hunk 3 above)
Comment (Peque) [inline src/main.c]: Done, thanks! :blush:

**Pull request merged**